<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClaimOffer</class>
 <widget class="QWidget" name="ClaimOffer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="wid" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget{
		 padding: 9px;
	     color: silver;
         background-image:url(:/icons/texture.png);
		  background-clip: border;

	     background-color: qlineargradient(spread:reflect,
                 x1:1, y1:0, x2:1, y2:1, stop:0 
                rgba(24, 25, 26, 190), stop:1 
                rgba(24, 25, 26, 250));

}
		</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="topMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">QLabel{
 background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;How to this works:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;1- You make a Youtube review video of application while you are in trail period,showing atleast 5 features of applications and installation instructions.&lt;/p&gt;&lt;p&gt;2- You need to include your account Id in the video description.&lt;/p&gt;&lt;p&gt;3- You fill the form below with your video ID, after review of your video your account gets activated following a confirmation email titled &amp;quot;Orion account activation&amp;quot;.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Email Id:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Video Id:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="video_id">
       <property name="placeholderText">
        <string>Youtube Video Id</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="account_id">
       <property name="text">
        <string>-</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Account Id:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="claimButton">
       <property name="text">
        <string>Claim</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icons/star-line.png</normaloff>:/icons/star-line.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="email_id">
       <property name="placeholderText">
        <string>Your email Id</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
